booking.c

```diff
@@ -712,6 +712,7 @@ void addBooking()
                 else
                 {
                     generateReferenceNumber(bookingReference, partial.booking.ticketID);
+
```

Comment:
?

Reply:
Just space added nothin more than that...